spec/models/user_spec.rb

```diff
@@ -349,4 +349,31 @@
       expect(nurse.district_level_sync?).to eq false
     end
   end
+
+  describe "flipperable" do
+    it "has a flipper_id" do
+      user = create(:user)
+
+      expect(user.flipper_id).to eq("User;#{user.id}")
+    end
+
+    it "has feature_enabled?" do
+      user = create(:user)
+      Flipper.enable(:a_flag, user)
+
+      expect(user.feature_enabled?(:a_flag)).to be true
+    end
+  end
+
+  describe "drug_stocks_enabled?" do
+    it "is true if any district accessible by the user is enabled" do
```

Comment:
I'd love to one more spec that flips on a facility _outside_ the admin's access, and ensures the result is false.